appsvc_fnc_dev_userssynch/Email.cs

```diff
@@ -0,0 +1,101 @@
+﻿using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+using Microsoft.Graph;
+using System;
+using System.Collections.Generic;
+using System.Net.Mail;
+using System.Net;
+using Azure.Identity;
+using Azure.Security.KeyVault.Secrets;
+
+namespace appsvc_fnc_dev_userssynch
+{
+    internal class Email
+    {
+        public static void SendEmail(string subject, string body, ILogger log)
+        {
+            IConfiguration config = new ConfigurationBuilder().AddJsonFile("appsettings.json", optional: true, reloadOnChange: true).AddEnvironmentVariables().Build();
+
+            string emailNotificationList = config["EmailNotificationListForUsersThatCannotBeInvited"];
```

Comment:
The list of email as to come from the storage account. Base on the department. For exemple, tbs will be from: TBS-To-GCX-B2B-Sync.json, DFO from: DFO-To-GCX-B2B-Sync.json, etc.